### PR TITLE
Remove net451 as a cross-compile target

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj [hunk 1]
```diff
@@ -17,7 +17,7 @@ Microsoft.Data.Sqlite.SqliteException
 Microsoft.Data.Sqlite.SqliteFactory
 Microsoft.Data.Sqlite.SqliteParameter
 Microsoft.Data.Sqlite.SqliteTransaction</Description>
-    <TargetFrameworks>net451;netstandard1.2</TargetFrameworks>
+    <TargetFrameworks>net46;netstandard1.2</TargetFrameworks>
```
Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: So you'll get netstandard on net45x? That's no good...

Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: Better stick with the min net version supported by netstandard 1.2

Comment (davidfowl) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: why?

Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: Why cripple net45x? Targeting netstandard1.2 means this library supports it one way or another...

Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: By cripple, I mean: without crosstargeting calling DbDataReader.Close() is a no-op on net45x

Comment (bricelam) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: dotnet/corefx#2249

Comment (pranavkm) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: cc @Eilon

Comment (Eilon) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: The lowest planned .NET Framework target is currently .NET 4.6.1. I think if we have strong evidence to suggest it's worth targeting anything lower than that, let's bring it up with @DamianEdwards .

Comment (davidfowl) [inline src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj]: > https://github.com/dotnet/corefx/issues/2249

Wow...

**Comment by bricelam** [comment]
Waiting on the outcome of dotnet/corefx#2249

**Pull request closed without merging**